app/apps/server/communication/uikit.js

```diff
@@ -14,6 +15,25 @@ const apiServer = express();
 
 apiServer.disable('x-powered-by');
 
+if (settings.get('API_Enable_CORS')) {
+	const CORSOriginSetting = settings.get('API_CORS_Origin');
+
+	const whitelistOrigins = CORSOriginSetting
```

Comment:
I'd suggest not using `blacklist/whitelist` names  and instead using the more "modern" approach to naming it as `blocklist/allowlist` 😉